spanner/spanner_snippets/spanner/spanner_create_table_with_datatypes.go

```diff
@@ -47,6 +47,7 @@ func createTableWithDatatypes(w io.Writer, db string) error {
 				LastContactDate DATE,
 				OutdoorVenue BOOL,
 				PopularityScore FLOAT64,
+				Revenue NUMERIC,
```

Comment:
Test failure:
```
integration_test.go:378: failed to add numeric column: rpc error: code = FailedPrecondition desc = Duplicate column name Venues.Revenue.
```

Reply:
The test failure has been fixed. It was caused by a different sample that adds the column, and that sample is also being executed as part of the integration tests.